tests/tests_pytorch/accelerators/test_hpu.py

```diff
@@ -76,6 +77,7 @@ def test_all_stages(tmpdir, hpus):
 
 
 @RunIf(hpu=True)
+@mock.patch.dict(os.environ, os.environ.copy())
```

Comment:
Had to set this. On this machine, somehow numpy/sklearn is setting these env vars for MKL:
'KMP_DUPLICATE_LIB_OK', 'KMP_INIT_AT_FORK' and our fixture detected the leakage. No big deal, we just make sure the env remains unchanged after the test.